pdc/apps/common/filters.py

```diff
@@ -50,6 +52,23 @@ def filter(self, qs, value):
         return qs
 
 
+class MultiValueRegexFilter(MultiValueFilter):
```

Comment:
yes. you are right. i am just thinking for different field, for example, component and role, & means "AND", but for component and component, & means "OR". i am just thinking if it is confused for the end user. Maybe this is my unnecessary concern. :)